geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaOutputFormat.scala

```diff
@@ -72,6 +73,56 @@ class GeoMesaOutputFormat extends OutputFormat[Text, SimpleFeature] {
 }
 
 /**
+  * Record reader that delegates to accumulo record readers and transforms the key/values coming back into
+  * simple features.
+  *
+  * @param reader
+  */
+class GeoMesaRecordReader[FI <: GeoMesaFeatureIndex[_, _, _]]
```

Comment:
yeah, I meant that the changes to this file and the GeoMesaAccumuloInputFormat were to abstract out the RecordReader, but then the abstractions weren't actually used I don't think. Seems like we should be able to revert the two files and not affect the hbase changes - if we were going to keep it we should move the abstract record reader into a GeoMesaInputFormat class (instead of here in the output format where it's out of place)